juristhese.csl

```diff
@@ -0,0 +1,174 @@
+            <number prefix=", p. " suffix="s." variable="page"/>
+          </else-if>
+          <else>
+            <text variable="container-title" font-style="italic" prefix="in " suffix=", "/>
+            <text macro="numero_container-title"/>
+          </else>
+        </choose>
+      </if>
+      <else-if type="book" match="any">
+        <choose>
+          <if match="any" variable="note">
+            <text variable="note" suffix=", "/>
+          </if>
+        </choose>
+        <choose>
+          <if match="any" variable="edition">
+            <text variable="edition" vertical-align="baseline"/>
+            <text value="e" vertical-align="sup" suffix=" "/>
+            <text value="éd." suffix=", "/>
+          </if>
+        </choose>
+        <text variable="publisher-place" suffix=", "/>
+        <text variable="publisher" suffix=", "/>
+        <date date-parts="year" form="text" variable="issued"/>
+        <names variable="contributor" prefix=", préf. "/>
+      </else-if>
+      <else-if type="article-magazine article-newspaper" match="any">
+        <choose>
+          <if match="any" variable="URL">
+            <text variable="container-title" quotes="false" font-style="italic" prefix="in " suffix=" (En ligne)"/>
+          </if>
+          <else>
+            <text variable="container-title" font-style="italic" prefix="in "/>
+          </else>
+        </choose>
+        <date form="text" variable="issued" prefix=", "/>
+      </else-if>
+      <else-if type="thesis" match="any">
+        <text value="thèse de doctorat" suffix=", "/>
+        <names variable="contributor" prefix="dir. " suffix=", "/>
+        <text variable="publisher" suffix=", "/>
+        <date date-parts="year" form="text" variable="issued"/>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="numero_container-title-short">
+    <choose>
+      <if match="any" variable="issue">
+        <text variable="issue" suffix="."/>
+        <text variable="page"/>
+      </if>
+      <else>
+        <date date-parts="year-month" form="text" variable="issued"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="numero_container-title">
+    <text variable="issue" prefix="n°" suffix=", "/>
+    <date date-parts="year-month" form="text" variable="issued"/>
+    <text variable="page" prefix=", p. "/>
+  </macro>
+  <citation>
+    <layout>
+      <choose>
+        <if match="any" position="ibid">
+          <text term="ibid" font-style="italic"/>
+        </if>
+        <else-if match="any" position="near-note">
+          <text macro="auteur"/>
+          <text macro="titre"/>
+          <text term="cited" font-style="italic"/>
+        </else-if>
+        <else>
+          <text macro="auteur"/>
+          <text macro="titre"/>
+          <text macro="edition"/>
+        </else>
+      </choose>
+    </layout>
+  </citation>
+  <bibliography>
+    <layout>
+      <date form="text" date-parts="year-month-day" variable="issued"/>
```

Comment:
this seems odd. 
If you don't have a bibliography I'd delete the whole block from lines 169-173, instead of just printing a date.

Reply:
My bad, it was supposed to be fixed ; it is now.